package.json

```diff
@@ -62,6 +62,7 @@
   "dependencies": {
     "long": "^5.2.0",
     "protobufjs": "^7.0.0",
+    "rxjs": "^7.8.0",
```

Comment:
Seems this is required in the newer version of the JS proto compiler. I was getting errors building the TS types without